modules/sor/sor.resolvers.ts

```diff
@@ -0,0 +1,13 @@
+import { Resolvers } from '../../schema';
+import { sorService as sorService } from './sor.service';
+import { tokenService } from '../token/token.service';
+
+const sorResolvers: Resolvers = {
```

Comment:
Since Beethoven and Balancer need to different return types for the sorGetSwaps, I'd suggest that we implement the resolvers under the Protocol specific resolvers in "modules/balancer" and "modules/beethoven"

Reply:
From there we can trigger two different methods for the SOR v1 and adapt SOR v2 to serve both formats